[2022-03-20 20:55:51,861] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-20 20:55:51,935] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-20 20:55:51,936] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:55:51,936] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2022-03-20 20:55:51,936] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:55:51,970] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): prepare_insert> on 2022-03-17T00:00:00+00:00
[2022-03-20 20:55:51,978] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_news', 'prepare_insert', '2022-03-17T00:00:00+00:00', '--job-id', '287', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/fetch_news.py', '--cfg-path', '/tmp/tmprttksy9m', '--error-file', '/tmp/tmproszvkp0']
[2022-03-20 20:55:51,975] {standard_task_runner.py:52} INFO - Started process 553 to run task
[2022-03-20 20:55:51,983] {standard_task_runner.py:77} INFO - Job 287: Subtask prepare_insert
[2022-03-20 20:55:52,123] {logging_mixin.py:104} INFO - Running <TaskInstance: fetch_news.prepare_insert 2022-03-17T00:00:00+00:00 [running]> on host e39215f0be5b
[2022-03-20 20:55:52,363] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=fetch_news
AIRFLOW_CTX_TASK_ID=prepare_insert
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T00:00:00+00:00
[2022-03-20 20:55:52,366] {python.py:118} INFO - Done. Returned value was: None
[2022-03-20 20:55:52,413] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=fetch_news, task_id=prepare_insert, execution_date=20220317T000000, start_date=20220320T205551, end_date=20220320T205552
[2022-03-20 20:55:52,502] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-20 20:55:52,569] {local_task_job.py:146} INFO - Task exited with return code 0
